sdk/agrifood/azure-agrifood-farming/README.md

```diff
@@ -44,6 +53,20 @@ client = FarmBeatsClient(endpoint="https://<my-account-name>.farmbeats.azure.net
 ```
 
 ## Key concepts
+Basic understanding of below terms will help to get started with FarmBeats client library.
+
+* [Farm Hierarchy][farm_hierarchy]: Is a collection of below entities.
```

Comment:
Generally for key concepts, we make each key concept an h3 level header. See [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/README.md#key-concepts) for an example.

I would recommend making an h3 for `Farm Hierarchy`, `Scenes`, and `Farm Operations`

Reply:
Fixed